RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+#0.1.3
```

Comment:
👍  updated release note for 0.1.2.

Reply:
Thanks @Tianhao-Gu .  I increased the version to v0.1.3 because there was a new commit with respect to the v0.1.2 that's released on CI (and it blocks re-release of the same version).  That implies there was another change after the first v0.1.2.  Anyone know what that change was?  Maybe I can figure it out from the commit logs, but I figure you all might know rather than me resort to code forensics.

Reply:
actually, it wasn't too bad.  The released commit on CI is df0bb6568008983c1daf6966cf8cebf54f0a0d80 which was from @qzzhang.  Ever commit after than is from @jsfillman and appears to be his cleanup of our github framework.  Not sure what that release note should be.  @jsfillman what should we say is the change in RELEASE_NOTES.md for v0.1.3?

Reply:
@qzzhang do you mind adding some release notes for https://github.com/kbaseapps/DataFileUtil/pull/67 ?

Reply:
@MrCreosote @Tianhao-Gu @jsfillman @qzzhang where are we at with this?  are the only changes still to record in the release notes the github stuff from @jsfillman and the colon to underscore for SHOCK names?

Reply:
@MrCreosote I went ahead and added what I could for the comments.  I'd really like to merge this so I can register it on CI.KBASE.US and do integration testing of client modules against it.  I'm sure @jsfillman would be better able to describe what changes he made between v0.1.2 and v0.1.3 (which I'm considering everything after commit df0bb65), but I made a much more general comment so we can get this through.  What do you think?

Reply:
I updated the release note for #67, then it created a new branch and pr #73.

Reply:
Also added release notes as requested in #74.

Reply:
merged your RELEASE_NOTES changes.  Working on errors in unit tests now.